terraform-sentinel/step2.md

```diff
@@ -0,0 +1,79 @@
+Now that you have created a policy, you will add some additional restrictions. 
+
+Open the policy file`terraform-sentinel/restrict-s3-buckets.sentinel`{{open}}, and add a print statement.
+
+## Create a print statement for debugging
+
+Your policy plan collection contains a lot of data. To see what data you are giving to your policy, create a print statement below the ending bracket your filter statement.
+
+```
+print(s3_buckets)
+```{{copy}}
+
+Run your Sentinel CLI apply again to see what data your plan contains.
+
+```
+sentinel apply -trace restrict-s3-buckets.sentinel
+```{{execute}}
+
+## Create a required tags variable
+
+Copy and paste the `required_tags` variable below your print statement. You are creating a list of variables that are required to be returned from the data you just generated in the previous print statement.
```

Comment:
Is the print statement required to use these values, or are we only printing to discover what values we can write policies against?

Reply:
This is for discovery and to understand how the collection is structured.

Reply:
Let's highlight that, and maybe have users remove it after they see the output once?

Does the `-verbose` command line flag also cause these values to print? Not sure why but the outputs feel a little bit busy to me 🤔

Reply:
verbose won't print the filter output, just the trace for each rule as it runs towards the main rule.